### PR TITLE
Button fo wiki page array of classes support

### PR DESCRIPTION
**Pull request by MasterMindNET**
Some styles were added to different screen sizes.
In JS you can add classes to which you want to append these buttons
Changelog updated.
```
// Object Consturctor:
// @position can be either default or fromRight side with % specifed in code 
function ParentElToAppend($parentEl, position = 'default') {
    this.$parentEl = $parentEl;
    this.position = position;
}

var $appendButtonTo = [
            new ParentElToAppend($('.layout-sidebar-container')),
            new ParentElToAppend($('.wiki-menu'), 'fromRight'),
            new ParentElToAppend($('#wallStream')),
]
// ...
// ...
// for the right side you can change this value it is set in %
var right = '5';
$btn.css({right: right + '%'});
```

There was a quesion considering these lines of code:
```
if (!$('#wallStream').length || !$sidebar.length) {
            return;
 }
```
Why do we have `!$('#wallStream').length` if it is never used in this file, Let me know, please!

### CODE REVIEW

**Comment by ArchBlood** [comment]
Thank you for the p/r, we'll look into this to see if this is something we want to merge into the repo.

**Comment by ArchBlood** [comment]
@buddh4 could you chime in on this seeing as it was your p/r that fixed the JavaScript?

**Comment by ArchBlood** [comment]
I'm not really seeing any benefits for Firefox users as `backdrop-filter` isn't fully supported and forces the user to enable it

https://caniuse.com/#feat=css-backdrop-filter

**Comment by MasterMindNET** [comment]
> I'm not really seeing any benefits for Firefox users as `backdrop-filter` isn't fully supported and forces the user to enable it
> 
> https://caniuse.com/#feat=css-backdrop-filter

Good day, @ArchBlood!

> `backdrop-filter` isn't fully supported and forces the user to enable it
Thank you for your advice!
Please, check if this looks good!
I cleaned up a few of my CSS changes.

Have a good day!

File: assets/js/scrollup.js [hunk 1]
```diff
@@ -7,19 +7,58 @@ humhub.module('scrollUpButton', function (module, require, $) {
...
+            var findFirstAcceptableElement = $checkInArr.find(el => checkIfExist(el.$parentEl));
+            if (!isSet(findFirstAcceptableElement) || !checkIfExist(findFirstAcceptableElement.$parentEl)) {
+                return null;
+            }
+            return findFirstAcceptableElement.$parentEl ? findFirstAcceptableElement.$parentEl : null
+        }
+
+        var $appendButtonTo = [
+            new ParentElToAppend($('.layout-sidebar-container')),
+            new ParentElToAppend($('.wiki-menu'), 'fromRight'),
```
Comment (ArchBlood) [inline assets/js/scrollup.js:39]: `fromRight` is ignored due to position setting.
https://github.com/GreenMeteor/humhub-scrollup-module/blob/46656b3709cff4c0dada0ea1dab99ecbfc56be84/models/ConfigureForm.php#L16

**Comment by ArchBlood** [comment]
> Why do we have `!$('#wallStream').length` if it is never used in this file, Let me know, please!

Sorry to get to this late but `$wallStream` is used on mobile from what I'm seeing in the `theme.css` file. :thinking:

**Comment by MasterMindNET** [comment]
> > Why do we have `!$('#wallStream').length` if it is never used in this file, Let me know, please!
> 
> Sorry to get to this late but `#wallStream` is used on mobile from what I'm seeing in the `theme.css` file.

All right, but nevertheless, I have added it to the array of classes, so I believe if it was working, it should work, I will test it next time, thank you for your business!

**Pull request merged**